CMakeLists.txt

```diff
@@ -82,10 +81,17 @@ add_executable(${PROJECT_NAME}
 )
 
 target_include_directories(ob-common
-    PRIVATE 
+    PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}/deps/
 )
 
+# Add GLM include directory
+target_include_directories(ob-common
```

Comment:
Don't think common needs glm as an include directory.

Reply:
Hello, since GLM is being included as `<glm/glm.hpp>` it is required to have glm as an include directory, if there's a different way to approach this, it'd be appreciated to know. The submodules should install in process of cloning the repository. Editing scripts is not required.

Reply:
Double checked, forgot that debug.h in common has glm as an include, my mistake sorry.